playbooks/openshift/delete-cluster.yml

```diff
@@ -4,6 +4,17 @@
 #  roles:
 #  - ../../galaxy/openshift-ansible-contrib/roles/openshift-pv-cleanup
 
+- hosts: localhost
+  vars_prompt:
+    name: "delete_cluster"
+    prompt: "Are you sure you want to delete this cluster? (yes/no)"
```

Comment:
Please add info on which cluster is about to be deleted - i.e.: cluster id or env_id.

Reply:
Added env_id to delete prompt.

cc/ @rohitmusti @procrypt